src/getting_started/installation_options.md

```diff
@@ -241,13 +241,15 @@ export PARSEC_SERVICE_ENDPOINT="unix:$(pwd)/parsec.sock"
 ```
 
 If parsec-tool is not installed into a directory included in `PATH`, then you also need to define
-`PARSEC_TOOL` environment variable with a full path to it:
+`PARSEC_TOOL` environment variable with a full path to it. Assuming the current working directory is
+the root folder of the `parsec-tool`:
 
 ```
-export PARSEC_TOOL="$(pwd)/parsec-tool"
+export PARSEC_TOOL="$(pwd)/target/debug/parsec-tool"
```

Comment:
I think the context is using the quick start bundle where we have the built binaries. 
So this change is not needed as it should be straight forward.

Reply:
The context here is:
`If parsec-tool is not installed into a directory included in PATH, then you also need to define PARSEC_TOOL environment variable with a full path to it:`

So that means that this is used to indicate how to do it if you are building or installing into a different path, no?